shared/ios/Keybase/AppDelegate.m

```diff
@@ -85,7 +85,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [self.window makeKeyAndVisible];
 
   // To simplify the cover animation raciness
-  self.resignImageView = [[UIImageView alloc] initWithFrame:self.window.bounds];
+  
+  // make a square of the biggest screen dimension to handle rotate snapshot
```

Comment:
i think this isn't actually what you want to do. i think you can just center the subview, something like
`self.resignImageView.center = [self.window convertPoint:self.window.center fromView:self.window.superview];`

Reply:
I tried this, it doesn't work.  Same problem.

Reply:
its likely cause its resizing itself based on the parent etc, anyways i don't htink this is a great fix but whatever